stable/jenkins/templates/config.yaml

```diff
@@ -43,6 +45,11 @@ data:
                   {{- $key }}={{ $value }}
                   {{- $_ := set $local "first" false }}
                 {{- end }}</nodeSelector>
+              <nodeUsageMode>NORMAL</nodeUsageMode>
```

Comment:
More config params that were not here before.

Reply:
Right. Will include in next commit.